exporters/prometheus/exporter.go

```diff
@@ -26,6 +26,13 @@ import (
 	"go.opentelemetry.io/otel/sdk/resource"
 )
 
+func init() {
+	// Ref: prometheus/common changed the default validation scheme to UTF-8 in v0.62.0
+	// https://github.com/prometheus/common/releases/tag/v0.62.0
+	// https://github.com/prometheus/client_golang/releases/tag/v1.21.0
+	model.NameValidationScheme = model.LegacyValidation
```

Comment:
Your original behavior was right there. We shouldn't rely on the legacy scheme by default. Especially since the changelog entry tells folks they can revert if they wish.

Reply:
I'll revert to that change then.

Reply:
Reverted to initial change, along with a mention in CHANGELOG.